Sources/Helpers/OAWikiImageHelper.h

```diff
@@ -15,6 +15,10 @@
 #define CM_LIMIT @"&cmlimit=500"
 #define FORMAT_JSON @"&format=json"
 #define IMAGE_BASE_URL @"https://commons.wikimedia.org/wiki/Special:FilePath/"
+#define USE_OSMAND_WIKI_API YES
+#define OSMAND_API_ENDPOINT @"https://osmand.net/api/"
```

Comment:
don't use #define. it is not type-safe. static NSString* const OSMAND_API_ENDPOINT = @"https://osmand.net/api/";